Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM circleci/node:10.14.1 as builder
+
+WORKDIR /usr/src/app
+
+COPY /src /usr/src/app/src
+COPY /public /usr/src/app/public
+COPY /package.json /usr/src/app/package.json
+COPY /tsconfig.json /usr/src/app/tsconfig.json
+COPY /yarn.lock /usr/src/app/yarn.lock
+
+RUN sudo chmod -R 777 /usr/src/app
+RUN yarn install
+RUN yarn build
+
+FROM nginx:1.16.1
+LABEL maintainer="https://teamdigitale.governo.it"
+
+COPY .env.io-onboarding-pa.development /usr/share/nginx/html/.env.io-onboarding-pa.development
```

Comment:
what's the point of having this file in the image?

Reply:
The env.sh file in this repo (copied as well into the container) has the goal of reading when the container starts environment variables and create a js file that is then read at runtime by the frontend application.
If the environment variables are not found, the bash script reads defaults from this file.

Reply:
If the environment variables are not found the container should not even start, you don't want to deploy a production container with development envs by mistake

Reply:
if the env file is missing, just print an error and exit

Reply:
What you express is one of the options. All our containers of IO start by default with development environment variables, and in case with production variables, if overwritten. The change would require to change the logic of the overall scripts (since we should anyway know what are the variables to grab from the environment - if any).
You can insert a story in the backlog to improve the change in the future.

Reply:
this is not one of the opinions, this is how things in this project are done :)

so make this image environment agnostic as all our docker images are